scst-store/deployment-details.hbs.md

```diff
@@ -20,6 +20,40 @@ The installation creates the following in your Kubernetes cluster:
 
 ## <a id='configuration'></a> Deployment configuration
 
+### Supported Network Configurations
+
+VMware recommended connection methods based on Tanzu Application Platform setup:
+
+* Single or multi-cluster with Contour = `Ingress`
+* Single cluster without Contour and with `LoadBalancer` support = `LoadBalancer`
+* Single cluster without Contour and without `LoadBalancer` = Port forwarding
```

Comment:
> without `LoadBalancer` = Port forwarding

I'm not sure I understand what this part means

Reply:
this was copied over from the previous doc, I should fix it a bit